vendor.conf

```diff
@@ -85,18 +85,25 @@ google.golang.org/grpc v1.0.5
 gopkg.in/check.v1 4f90aeace3a26ad7021961c297b22c42160c7b25
 gopkg.in/inf.v0 v0.9.0
 gopkg.in/yaml.v2 f7716cbe52baa25d2e9b0d0da546fcf909fc16b4
-k8s.io/apiextensions-apiserver be41f5093e2b05c7a0befe35b04b715eb325ab43
-k8s.io/apimachinery release-1.7
-k8s.io/client-go v4.0.0
+k8s.io/apiextensions-apiserver kubernetes-1.8.2
+k8s.io/apimachinery kubernetes-1.8.2
+k8s.io/api kubernetes-1.8.2
+k8s.io/client-go kubernetes-1.8.2
 k8s.io/kubernetes v1.5.1
 github.com/sasha-s/go-deadlock master
 github.com/petermattis/goid 0ded85884ba5c4c9267fcd5a149061f7c3455eee
 github.com/google/gops 25312cafb9a191a6d377c480a4666beec3105e4a
 github.com/kardianos/osext ae77be60afb1dcacde03767a8c37337fad28ac14
 github.com/kr/text 7cafcd837844e784b526369c9bce262804aebc60
 github.com/kr/pretty cfb55aafdaf3ec08f0db22699ab822c50091b1c4
-github.com/optiopay/kafka 02fdf3fd241af9fe0b226577627e24b7f9a4c8e5 https://github.com/cilium/kafka
+github.com/optiopay/kafka 41e2198304524d5f0e0f47724832bcd385229902 https://github.com/cilium/kafka
```

Comment:
Why are you updating kafka in this commit?

Reply:
02fd no longer exists, I had to update it to 41e21 which is the last one in `https://github.com/cilium/kafka`